### PR TITLE
Add <auc:RoofCondition> element to <auc:RoofID> element

### PR DESCRIPTION
**Pull request by markborkum**
See `proposals/2019/Add RoofCondition to RoofSystemType.md` for details.

Merge after #116

### CODE REVIEW

**Comment by jasondegraw** [comment]
@nllong @markborkum The condition of the roof belongs with the link to the roof, not within the roof description. This approach means that different roof system elements are needed each time a different conditions, so I'm not for merging this. Std 211 needs support for something similar, so a solution is needed, but timing could be an issue.

**Comment by markborkum** [comment]
@nllong In Audit Template, roofs are associated with buildings and not subsections, so we are not able to use the `<auc:RoofID>` element in this way.

**Comment by jasondegraw** [comment]
@markborkum I'm not sure what "we are not able to use the `<auc:RoofID>` element in this way" means, but as I indicated there's a solution needed for Std 211.

File: BuildingSync.json [hunk 1]
```diff
@@ -58941,6 +58941,29 @@
 					},
 					"type": "object"
 				},
+				"auc:RoofCondition": {
```
Comment (nllong) [inline BuildingSync.json]: The JSON schema will be regenerated when we release 2.0-prerelease.

**Comment by nllong** [comment]
@markborkum can you review this one again since you created the PR.

**Comment by markborkum** [comment]
@nllong I approve the changes. (Authors cannot review their own pull requests.)

**Pull request merged**